src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionGridStyles.kt

```diff
@@ -23,6 +25,50 @@ class CollectionGridStyles: Stylesheet() {
 
         contentLoadingProgress {
             progressColor = AppTheme.colors.appRed
+            backgroundColor += Color.WHITE
```

Comment:
I noticed we have other colors defined in AppTheme.colors. Should they all be there or can I directly use Color.White here? (Same question applies below.)

Reply:
I think it'd be better to have it in AppTheme. If there is a change to the theme, and this was Color.WHITE the change to theme wouldn't be able to change it. so yeah apptheme

Reply:
done